src/app/plugins/connectfour.plugin.ts

```diff
@@ -351,9 +367,14 @@ class ConnectFourGame {
     );
   }
 
-  private async _updateGameState(msg: IMessage): Promise<void> {
+  private async _updateGameState(msg: IMessage, timedOut: boolean): Promise<void> {
```

Comment:
we can make the second argument be optional and have it as an object. either named as `options` or `params` and have `timedOut` be an optional property in this object

this will allow you to simplify the invocation for consumers of this function that aren't coming from a `timedOut` source